src/js/test/index.test.js

```diff
@@ -0,0 +1,92 @@
+import TestUtils from 'js-test-buddy';
+import { getBreakpoints, getCurrentScreenWidth } from '../';
+
+describe('getBreakpoints', () => {
+
+    beforeEach(() => {
+        TestUtils.setBodyHtml(`
+        <style>
+            .c-screen-sizer {
+                content: 'narrow:414px,mid:768px,wide:1025px,huge:1280px';
+            }
+        </style>
+        `);
+    });
+
+    it('should return an object', () => {
+        // Act
+        const breakpoints = getBreakpoints();
+
+        // Assert
+        expect(typeof breakpoints).toBe('object');
+    });
+
+    it('should return four breakpoints', () => {
+        // Act
+        const breakpoints = getBreakpoints();
+
+        // Assert
+        expect(Object.entries(breakpoints).length === 4);
```

Comment:
`expect(Object.entries(breakpoints)).toBe(4)`

Reply:
Jinx! 👾

Reply:
@kevinrodrigues `expect(Object.entries(breakpoints).length).toBe(4)` ?

Reply:
Sorry missed the `.length` :)